projects/sd-jwt-rs.md

```diff
@@ -0,0 +1,83 @@
+# Project Name
+
+SD-JWT Rust Reference Implementation
+
+# Project Description
+
+This is the reference implementation of the [IETF SD-JWT specification](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) written in Rust.
+
+# Alignment with the OpenWallet Foundation Mission
+
+This library implements the Selective Disclosure for JWTs specification, which is an important building block for future digital identity wallets.
+Rust basis may be used as a source for high-performance wallets as well as a universal starting point for native platform-specific implementation for both mobile and desktop.
+
+# Code of Conduct
+
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)
+
+# TAC Sponsor
+
+To be defined.
+
+# Project License
+
+Apache 2.0
+
+# Source Control
+
+To be defined.
+Please note: this is a brand-new source code. The DSR team is working on the initial implementation and may append this section to the nearest feature.
```

Comment:
Just to confirm, the expectation is that this would be in GitHub (specifically under https://github.com/openwallet-foundation-labs). Is that correct?

Reply:
Correct. We started with some private repos, but the end goal (as soon as approved) is to make the lab location the main one.

Reply:
Is there a reason the repository is private? Are there external dependencies, and if yes, what are their licenses?